src/Disks/IO/ReadBufferFromAzureBlobStorage.h

```diff
@@ -50,14 +50,12 @@ class ReadBufferFromAzureBlobStorage : public ReadBufferFromFileBase
 
 private:
 
-    void initialize();
+    size_t readBytes();
```

Comment:
renamed this function, as it also reads data now. But we still need `initialized` bool for other checks, so kept that as it is.